src/main/java/com/shanebeestudios/skbee/elements/bound/conditions/CondBoundIsTemporary.java

```diff
@@ -0,0 +1,30 @@
+package com.shanebeestudios.skbee.elements.bound.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import com.shanebeestudios.skbee.elements.bound.objects.Bound;
+
+@Name("Bound - Is Temporary")
+@Description("Check if a bound is temporary.")
+@Examples("if bound with id \"temporary-bound\" is temporary:")
+@Since("INSERT VERSION")
+public class CondBoundIsTemporary extends PropertyCondition<Bound> {
+
+    static {
+        register(CondBoundIsTemporary.class, "temporary", "bound");
```

Comment:
I do worry about this syntax, potentially colliding with something in the future.
Maybe `"[a] temporary bound"` .... so the example would be:
`if bound with id "test-bound" is a temporary bound:`

Reply:
```suggestion
        register(CondBoundIsTemporary.class, "[a] temporary bound", "bound");
```

To make it a little easier if so

Reply:
I reviewed on how SkBee is set up and attempts to adapt to it, and there aren't any syntax which supports plurals.
If any of you would like to have this condition support plural, do let me know

Reply:
maybe `"[a] temporary bound[s]"`

Reply:
So plural it is!

Reply:
If no plurals are supported rn it's something we should think about updating to support